spec/ImgixTagSpec.js

```diff
@@ -152,6 +153,19 @@ describe('ImgixTag', function() {
     });
   });
 
+  it('includes the `ixjsv` parameter when `imgix.config.includeLibraryParam` is `true`', function() {
```

Comment:
Should we also have a test asserting that this config setting is `true` by default?